pyvista/plotting/renderer.py

```diff
@@ -1455,7 +1465,34 @@ def disable_eye_dome_lighting(self):
         self.edl_pass.ReleaseGraphicsResources(self.parent.ren_win)
         del self.edl_pass
         self.Modified()
-        return
+
+    def enable_shadows(self):
```

Comment:
Return just isn't required here, that's all.  `Modified` may or may not be required, but I don't think it hurts in this case.

Reply:
Sorry, I meant the entirety of `enable_shadows` and `disable_shadows`. I don't understand VTK enough (=at all when shadows are concerned). Just wanted to note that I can't review the VTK bits because I don't understand them.

(The changes in the preview above are fine, I just attached my comment to `def enable_shadows`)